configs/example-config.yaml

```diff
@@ -0,0 +1,4 @@
+mongo_url: "mongodb://10.1.15.231:27017"
```

Comment:
lets replace `default-config.yaml` with this `example-config.yaml` we don't need both.

Reply:
just removed it